bigtable/client/client_options.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+#include "bigtable/client/client_options.h"
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+
+ClientOptions::ClientOptions() {
```

Comment:
Maybe we should have a free standing function (or static function) called `EmulatorClientOptions()` that returns a properly configured client options to use with the emulator?  No need to change this PR, just file an issue and add a TODO() linking it.

Reply:
Created New Issue.